airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -3065,6 +3069,7 @@ def execute(self, context: Context):
                         project_id=self.project_id,
                         region=self.region,
                         gcp_conn_id=self.gcp_conn_id,
+                        tags=self.tags,
```

Comment:
In my opinion we shouldn't pass tags to the trigger because the only thing it is responsible for - is polling the batch by batch_id. Because the batch_id is unique and sufficient for the batch identification, we probably shouldn't use tags here. Moreover, this PR doesn't contain any changes for the `DataprocBatchTrigger`, so it doesn't even accept this parameter.